.tekton/timestamp-authority-pull-request.yaml

```diff
@@ -95,11 +95,11 @@ spec:
       description: Skip optional checks, set false if you want to run optional checks
       name: skip-optional
       type: string
-    - default: "false"
+    - default: "true"
       description: Execute the build with network isolation
       name: hermetic
       type: string
-    - default: ""
+    - default: [{"path": ".", "type": "gomod"}, {"path": "./hack/tools", "type": "gomod"}]
```

Comment:
Instead of changing these values in `spec.pipelineSpec.params` they should be added to `spec.params` on line 20. This is something I dug into with RHTAP team this week and the docs were just updated to make it clearer. E.g. see the *Note* here https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/proc_hermetic-builds/